From 119e52519cc50a34e99b60524eca856e0e0a835f Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Wed, 14 Nov 2012 23:52:34 +0200
Subject: [PATCH 04/11] [#3582257]"Open entered folder" button do nothing

This should be done another way (as it is done in libfm-demo)
since until we activate entry the path in it isn't updated yet.
---
 src/main-win.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main-win.c b/src/main-win.c
index 4431106..b3e462a 100644
--- a/src/main-win.c
+++ b/src/main-win.c
@@ -903,7 +903,7 @@ static void on_close_tab(GtkAction* act, FmMainWin* win)
 
 static void on_go(GtkAction* act, FmMainWin* win)
 {
-    fm_main_win_chdir(win, fm_path_entry_get_path(win->location));
+    fm_main_win_chdir_by_name(win, gtk_entry_get_text(GTK_ENTRY(win->location)));
 }
 
 static void _update_hist_buttons(FmMainWin* win)
-- 
1.8.0.1

